xml/bp_troubleshooting_osds.xml

```diff
@@ -759,13 +773,29 @@ osd.0 is down since epoch 23, last address 192.168.106.220:6800/11080
    you may check if the disk where the OSD is located is mounted:
   </para>
 <screen>&prompt.root;lsblk -f
+<!--    the output has changed a bit:-->
```

Comment:
@jschmid1 could you provide an updated output?

Reply:
uch, I actually pasted it below this comment.. The formatting sucks though. Re-pasting:

```

dev-box-1:~ # lsblk -f
NAME                                                                                                  FSTYPE      LABEL UUID                                   FSAVAIL FSUSE% MOUNTPOINT
vda
├─vda1
├─vda2                                                                                                vfat        EFI   7BDD-40C3                                18.8M     6% /boot/efi
└─vda3                                                                                                ext4        ROOT  144d3eec-c193-4793-b6a9-1ac295259f4b     36.7G     5% /
vdb                                                                                                   LVM2_member       Fj8nlY-Dnmm-8Y0w-tJrO-8rAe-SUTH-lA2Ux2
└─ceph--6dc6f71f--ff02--4316--b145--c7804d5fb2f4-osd--block--2cb16f65--c87d--405d--a913--4e4ea6037ca1
vdc                                                                                                   LVM2_member       0101m1-mc83-K8ce-j4Dv-yxUO-5KPj-6FpuSu
└─ceph--1082eac0--4478--48cf--b8ab--4d3a62ca1c27-osd--data--185c9dd6--ce28--4b98--a9b6--9d5b1f444a4f
vdd                                                                                                   LVM2_member       wqE8sC-w5mx-J9t1-FoM1-vIRC-0xxq-5FPyuL
└─ceph--9e71d81a--e394--49b7--bef9--b639083be779-osd--data--eb94f8e6--af6d--4ef6--911b--7f44f7484c85


```

Reply:
I see! That did format oddly. Thanks!